types/node/test/crypto.ts

```diff
@@ -30,6 +30,11 @@ import { promisify } from 'util';
         .update(new Int8Array(new Buffer('world').buffer)).digest('hex');
 }
 
+{
+    const hashResult: string = crypto.createHash('md5')
+        .update('world', 'utf16le').digest();
```

Comment:
The type is also used in `update()` of Hmac, Signer, Verify so we should add tests for those as well in my opinion